ocp_install_env.sh

```diff
@@ -95,6 +95,11 @@ controlPlane:
 platform:
   baremetal:
     dnsVIP: ${DNS_VIP}
+    provisioningInterface: "ens3"
+    provisioningNetworkCIDR: "172.22.0.0/24"
+    provisioningIP: "172.22.0.3/24"
+    provisioningDHCPRange: "172.22.0.10,172.22.0.100"
+    rhcosImageURL: "${RHCOS_IMAGE_URL}"
```

Comment:
As mentioned on the installer PR we shouldn't add rhcosImageURL because it's data that's already known to the installer - we only care about RHCOS_IMAGE_URL in dev-scripts because we cache the image